Minio.Tests/UtilsTest.cs

```diff
@@ -89,7 +89,7 @@ public void TestVeryLongObjectName()
             {
                 Assert.AreEqual(ex.message, "Object name cannot be greater than 1024 characters.");
             }
-            
```

Comment:
Can revert this unexpected change

Reply:
Sorry,I don't know why it remove the space automatically 😢

Reply:
![image](https://user-images.githubusercontent.com/7550366/65408059-dc689900-de16-11e9-89e1-9e7317d59f4c.png)

Reply:
it is your editor/linter that is producing this. No problem. You can use vi/emacs/nano/gedit to remove this

Reply:
Ok, I fixed it now